Cache/TreeCache.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace Becklyn\RouteTreeBundle\Cache;
+
+use Becklyn\RouteTreeBundle\Tree\Node;
+use Symfony\Component\HttpKernel\HttpKernel;
+use Symfony\Component\HttpKernel\KernelInterface;
+
+
+/**
+ *
+ */
+class TreeCache
+{
+    /**
+     * @var bool
+     */
+    private $isDebug;
+
+    /**
+     * @var string
+     */
+    private $filePath;
+
+
+    /**
+     * @param KernelInterface $kernel
+     */
+    public function __construct (KernelInterface $kernel)
```

Comment:
I'd rather have two distinct parameters for the cache base dir and the debug parameter instead of taking a hard dependency on the `KernelInterface` (easier for unit testing and I don't like the dependency to the kernel itself).